.github/workflows/protocol-pregenesis.yml

```diff
@@ -0,0 +1,33 @@
+name: Protocol Pregenesis
+
+on:  # yamllint disable-line rule:truthy
+  pull_request:
+    paths:
+      - 'protocol/scripts/genesis/**'
+      - 'protocol/testing/**'
+  push:
+    branches:
+      - main
+      - 'release/protocol/v0.[0-9]+.x'  # e.g. release/protocol/v0.1.x
```

Comment:
for my understanding, are we going to use the letter `x`? I thought it was just a placeholder

Reply:
The branch name explicitly has letter `x`, like `release/protocol/v1.x`